tethys_gizmos/urls.py

```diff
@@ -15,15 +15,17 @@
     url(r'^get-kml/$', gizmo_showcase_views.get_kml, name='get_kml'),
     url(r'^swap-kml/$', gizmo_showcase_views.swap_kml, name='swap_kml'),
     url(r'^swap-overlays/$', gizmo_showcase_views.swap_overlays, name='swap_overlays'),
+    url(r'^(?P<job_id>[\d.@+-]+)/terminate', jobs_table_views.terminate, name='terminate_job'),
     url(r'^(?P<job_id>[\d.@+-]+)/delete$', jobs_table_views.delete, name='delete_job'),
     url(r'^(?P<job_id>[\d.@+-]+)/resubmit$', jobs_table_views.resubmit, name='resubmit_job'),
     url(r'^(?P<job_id>[\d.@+-]+)/show-log', jobs_table_views.show_log, name='show_log'),
+    url(r'^(?P<job_id>[\d.@+-]+)/log-content/(?P<key1>[\w+_-]+)(?:/(?P<key2>.*))?', jobs_table_views.get_log_content,
+        name='log_content'),
     url(r'^(?P<job_id>[\d.@+-]+)/execute', jobs_table_views.execute, name='execute_job'),
     url(r'^(?P<job_id>[\d.@+-]+)/update-row', jobs_table_views.update_row, name='update_job_row'),
     url(r'^(?P<job_id>[\d.@+-]+)/update-workflow-nodes-row', jobs_table_views.update_workflow_nodes_row,
         name='update_workflow_nodes_row'),
     url(r'^(?P<job_id>[\d.@+-]+)/(?P<type>[\w-]+)/insert-bokeh-row', jobs_table_views.bokeh_row, name='bokeh_row'),
-    url(r'^(?P<job_id>[\d.@+-]+)/update-status', jobs_table_views.update_status, name='update_job_status'),
```

Comment:
Right. There were several methods handling update-status, but it wasn't being used. I think I went back and forth early on because I couldn't decide which approach to do, but I think we're pretty firmly in the `update-row` camp now, so I've removed all of the unused code.